crypto/key_store_passphrase.go

```diff
@@ -112,40 +113,44 @@ func (ks keyStorePassphrase) GetKey(keyAddr []byte, auth string) (key *Key, err
 	return key, err
 }
 
-func (ks keyStorePassphrase) GetKeyAddresses() (addresses [][]byte, err error) {
+func (ks keyStorePassphrase) GetKeyAddresses() (addresses []common.Address, err error) {
 	return GetKeyAddresses(ks.keysDirPath)
 }
 
 func (ks keyStorePassphrase) StoreKey(key *Key, auth string) (err error) {
 	authArray := []byte(auth)
-	salt := randentropy.GetEntropyMixed(32)
+	salt := randentropy.GetEntropyCSPRNG(32)
```

Comment:
Looking at upstream packages, it seems as though the windows implementation will return 0 if CryptGenRandom random fails. I can't tell from packages if this is properly handled upstream, however, I doubt it because GetEntropyCSPRNG doesn't return an error type. Therefore, we should be checking that salt isn't all zeros.
TODO: Determine what happens when upstream RNG fails -- CryptGenRandom can fail on windows and /dev/random maybe unavailable -- Can this be detected? What happens? How to handle this properly?